docs/en/simulation/hitl.md

```diff
@@ -60,32 +59,48 @@ In summary, HITL runs PX4 on the actual hardware using standard firmware, but SI
 
 ## Setting up HITL
 
-### PX4 Configuration
+## Add HITL to Firmware
```

Comment:
So here, pulling the HITL setup out too.

Do we really only need `pwm_out_sim` for HITL?

Reply:
Yes, HITL only needs that module as all the sensors are handled by the simulator/mavlink interface